src/components/routing/routers/AppRouter.js

```diff
@@ -28,6 +28,7 @@ const AppRouter = ({ stompWebSocketHook }) => {
             <Routes>
 
                 <Route path="/lobbyoverview/*" element={<LobbyOverviewRouter base="/lobbyoverview" stompWebSocketHook={stompWebSocketHook} />} />
+                <Route path="/lobbyoverview/*" element={<Navigate to ="/lobbyoverview" />} />
```

Comment:
This statement does not seem to have an effect

Reply:
that's true, i removed it